File: clean-master-web-app/src/java/VirusScanner.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class VirusScanner {
    public static void scanForVirus(String filename) {
        String virusSignature = "Trojan1234";
        try (BufferedReader reader = new BufferedReader(new FileReader(filename))) {
            String line;
            while ((line = reader.readLine()) != null) {
                if (line.contains(virusSignature)) {
                    System.out.println("Virüs tespit edildi: " + virusSignature);
                }
            }
        } catch (IOException e) {
            System.out.println("Dosya açılmadı.");
        }
    }

    public static void main(String[] args) {
        scanForVirus("test_file.txt");
    }
}

File: clean-master-web-app/src/rust/virus_scanner.rs
use std::fs::File;
use std::io::{self, BufRead};
use std::path::Path;

fn scan_for_virus(filename: &str) {
    let virus_signature = "Trojan1234";
    let file = File::open(filename).expect("Dosya açılamadı");
    let reader = io::BufReader::new(file);

    for line in reader.lines() {
        let line = line.expect("Satır okunamadı");
        if line.contains(virus_signature) {
            println!("Virüs tespit edildi: {}", virus_signature);
        }
    }
}

fn main() {
    scan_for_virus("test_file.txt");
}

